lib/fluent/plugin/out_http_shadow.rb

```diff
@@ -24,7 +24,7 @@ def initialize
     config_param :username, :string, :default => nil
     config_param :password, :string, :default => nil
     config_param :rate, :integer, :default => 100
-    config_param :rate_per_method_hash, :hash, :default => nil
+    config_param :rate_per_host_hash, :hash, :default => nil
```

Comment:
`rate_per_method_hash ` is our original option. ref: #1 
I think that if the `1. ~` situation occurrs we can rebase.